i18n/en-US.properties

```diff
@@ -1276,6 +1276,8 @@ boxui.share.sharedLinkSettings.customURLLabel = Non-private custom URL
 boxui.share.sharedLinkSettings.directDownloadBlockedByAccessPolicyWithClassification = Download has been disabled for content due to the classification
 # Text to show that direct link download is disabled due to applied shield access policy without classification
 boxui.share.sharedLinkSettings.directDownloadBlockedByAccessPolicyWithoutClassification = Download has been disabled for content without classification.
+# Text to show that direct link download is disabled due to content marked as malicious content
```

Comment:
Minor: Remove "content marked as"?

Also, should the text have a period at the end?

Reply:
👍